### PR TITLE
Obtaining output tensor size from OpticalFlowAdapter

### PR DESCRIPTION
**Pull request by szalpal**

### CODE REVIEW

File: dali/aux/optical_flow/turing_of/optical_flow_turing_test.cc [hunk 1]
```diff
@@ -163,37 +163,41 @@ TEST(OpticalFlowTuringTest, DISABLED_CalcOpticalFlowTest) {
...
-
 
   OpticalFlowParams params = {0.0, VectorGridSize::SIZE_4, false};
   try {
     OpticalFlowTuring of(params, width, height, channels);
+
+    // Output
+    auto out_shape = of.GetOutputShape().to_static<3>();
+    auto memout = kernels::memory::alloc_unique<float>(kernels::AllocType::Unified,
+                                                       static_cast<size_t>(std::accumulate(
```
Comment (Kh4L) [inline dali/aux/optical_flow/turing_of/optical_flow_turing_test.cc]:
```suggestion
                                                       volume(out_shape)
```

Comment (szalpal) [reply dali/aux/optical_flow/turing_of/optical_flow_turing_test.cc]: Done

**Comment by szalpal** [comment]
#680034

**Comment by szalpal** [comment]
DALI-340

**Pull request merged**